user_acceptance_tests/tests/happy.rb

```diff
@@ -16,17 +16,17 @@
   end
 
   it 'GET /api/inventory/1 should return HTTP 200 OK' do
-    response = RestClient.get("#{service_url}/api/inventory")
+    response = RestClient.get("#{service_url}/api/inventory/1")
     expect(response.code).must_equal(200)
   end
 
   it 'GET /api/validateMessage should return HTTP 200 OK' do
-    response = RestClient.get("#{service_url}/api/inventory")
+    response = RestClient.get("#{service_url}/api/inventory/validateMessage")
```

Comment:
I think this should be `/api/validateMessage` and the behavior one should be `/api/behaviors`